packages/orchestration/src/orchestration-api.ts

```diff
@@ -12,7 +12,15 @@ import type {
 } from '@agoric/ertp/src/types.js';
 import type { LocalChainAccount } from '@agoric/vats/src/localchain.js';
 import type { Timestamp } from '@agoric/time';
-import type { IBCMsgTransferOptions, KnownChains } from './types.js';
+import type {
+  ChainInfo,
+  CosmosChainAccountMethods,
+  CosmosChainInfo,
+  IBCMsgTransferOptions,
```

Comment:
This file shouldn't depend on cosmos types, should it?

Reply:
ideally not, but it already does. I see that as independent cleanup